### PR TITLE
fileNo field added

### PR DESCRIPTION
**Pull request by git4saurabh**
guys  I have added a field called fileNo please check

### CODE REVIEW

**Review by KanchanAnkushe — APPROVED** [review]

**Review by Onkar2991 — APPROVED** [review]
chan saurabh

File: src/main/java/com/fileUploadProject/payload/FileResponse.java [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 
-   private String fileName;
-   private String message;
-   private String fileSize;
-
-   private String fileType;
+    private String fileName;
+    private String message;
+    private String fileSize;
+    private int fileNo;
```
Comment (ShaileshWattamwar) [inline src/main/java/com/fileUploadProject/payload/FileResponse.java:8]: @git4saurabh **don't make more complex code**

**Review by ShaileshWattamwar — APPROVED** [review]
Make more durable code rather than making it more complex

**Review by VishalZerikunthe — APPROVED** [review]
approved

**Review by dholeakshay — APPROVED** [review]
yeah

**Pull request merged**